grails-app/services/com/unifina/service/ProductService.groovy

```diff
@@ -64,6 +64,9 @@ class ProductService {
 
 	Product create(CreateProductCommand command, SecUser currentUser)
 			throws ValidationException, NotPermittedException {
+		if (command.name == null || command.name.trim() == "") {
```

Comment:
Similar comment here

Reply:
CreateProductCommand can contain an empty name. So empty name is replaced with the default.